envFiles/.env.rootless.production

```
@@ -0,0 +1,148 @@
+API_REDIS_HOST=redis
+API_REDIS_PORT=6379
+# Optional: Override default cache TTLs per entity (in seconds). Format: JSON object.
+# CACHE_ENTITY_TTLS={"user": 600, "organization": 600, "event": 240, "post": 120, "actionItem": 120}
+API_GRAPHQL_SCALAR_FIELD_COST=0
+API_GRAPHQL_SCALAR_RESOLVER_FIELD_COST=1
+API_GRAPHQL_OBJECT_FIELD_COST=1
+API_GRAPHQL_LIST_FIELD_COST=1
+API_GRAPHQL_NON_PAGINATED_LIST_FIELD_COST=5
+API_GRAPHQL_MUTATION_BASE_COST=10
+API_GRAPHQL_SUBSCRIPTION_BASE_COST=15
+API_RATE_LIMIT_BUCKET_CAPACITY=500  
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)